frontend/src/utilities/notebookControllerUtils.ts

```diff
@@ -310,7 +312,8 @@ export const getNotebookEventStatus = (
 ): NotebookProgressStep => {
   const timestamp = new Date(getEventTimestamp(event)).getTime();
 
-  if (event.message.includes('oauth-proxy')) {
+  // For Oauth-related events
+  if (event.message.includes('oauth-proxy') || event.message.includes('ose-oauth-proxy')) {
```

Comment:
Funnily enough -- this change actually didn't do anything. It doesn't hurt to keep it -- but my bad (again lol)